efficientnet/src/config.ts

```diff
@@ -0,0 +1,1056 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+export const config = {
+  BASE_PATH: 'https://storage.googleapis.com/gsoc-tfjs/models/efficientnet',
+  CROP_PADDING: 32,
+  CROP_SIZE: {
+    b0: 224,
+    b1: 240,
+    b2: 260,
+    b3: 300,
+    b4: 380,
+    b5: 456,
+  },
+  MEAN_RGB: [0.485, 0.456, 0.406],
+  STDDEV_RGB: [0.229, 0.224, 0.225],
+  IMAGENET_CLASSES: {
```

Comment:
I think we should move these classes to a separate JSON file and import it here.

Reply:
This seems to be more natural, since otherwise we have to cast class labels to integers, losing the information stored in the data itself.